cluster/manifests/skipper/deployment.yaml

```diff
@@ -320,9 +320,21 @@ spec:
           - "-forwarded-headers=X-Forwarded-For,X-Forwarded-Proto=https,X-Forwarded-Port=443"
           - '-forwarded-headers-exclude-cidrs={{ .cluster_internal_cidrs | join "," }}'
 {{ end }}
-{{ if .Cluster.ConfigItems.skipper_ingress_inline_routes }}
-          - "-inline-routes={{ .Cluster.ConfigItems.skipper_ingress_inline_routes }}"
-{{ end }}
+          - "-inline-routes"
+          - |
+            kube__healthz_down: Path("/kube-system/healthz")
+              && Shutdown()
+              && SourceFromLast("10.0.0.0/8", "192.168.0.0/16", "172.16.0.0/12", "127.0.0.1/8", "fd00::/8", "::1/128")
```

Comment:
I think for EKS clusters we are missing WhitelistedIPs
https://github.com/zalando-incubator/kubernetes-on-aws/blob/18cb6e0afd46c6244d06ea79fd98fd81a0396b3f/cluster/manifests/skipper/deployment.yaml#L556

in skipper we append those IPs to a global value 

https://github.com/zalando/skipper/blob/15385fae75b3e384d6bed671266d4a8906e055fa/dataclients/kubernetes/kube.go#L309

Maybe that's why EKS clusters are failing

Reply:
This was done in https://github.com/zalando-incubator/kubernetes-on-aws/pull/9206